src/fidesops/main.py

```diff
@@ -40,6 +42,75 @@
         allow_headers=["*"],
     )
 
+
+@app.middleware("http")
+async def dispatch_log_request(request: Request, call_next: Callable) -> Response:
+    fides_source: Optional[str] = request.headers.get("X-Fides-Source")
```

Comment:
If you're calling the API directly, I assume `fides_source` is None - is that how you're separating the data in fideslog?  You just assume if there's no fides_source, then it's an API call?

Reply:
correct, the only way we can discern direct api call vs not is by setting specific source headers from our clients.